src/edge_proxy/settings.py

```diff
@@ -133,6 +133,7 @@ class AppSettings(BaseModel):
     logging: LoggingSettings = LoggingSettings()
     server: ServerSettings = ServerSettings()
     health_check: HealthCheckSettings = HealthCheckSettings()
+    proxy: Optional[str] = None
```

Comment:
Just due to the fact that this component is already called the 'Edge Proxy', I think we should more explicit with the naming of this setting. Can we update it to `request_client_proxy` ? We can obviously add it to the documentation on our side too to make it clear, but I think it would be good to be slightly more explicit with the setting. 

Let me know if you disagree.

Reply:
Also, for consistency (and to follow modern typing best practices), can we change the typing to `str | None`. 

Then, based on these 2 comments together, we should end up with something like:

```suggestion
    request_client_proxy: str | None = None
```

Reply:
@matthewelwell I'd like to improve upon your comment and propose to just namespace the setting, akin to the logging settings we currently have.

The namespace could be called `httpx` or `http_client`.

Reply:
Additionally, I expect a docs PR to the main repo.